core/src/main/resources/jenkins/security/whitelisted-classes.txt

```diff
@@ -1,11 +1,25 @@
+com.google.common.collect.AbstractListMultimap
+com.google.common.collect.AbstractMultimap
```

Comment:
Should solve [JENKINS-49016](https://issues.jenkins-ci.org/browse/JENKINS-49016) AFAICT.

`HashMultimap` was already whitelisted, as was its superclass `AbstractSetMultimap`, but not _its_ (`Serializable`) superclass `AbstractMultimap`. I suppose the `XStream2.BlacklistedTypesConverter` only cares about the concrete type of the object being saved, whereas Java serialization pays attention to the whole (`Serializable`) inheritance chain.

Reply:
Yep, we may have to implicitly whitelist class hierarchies in ClassFilter. Will create a ticket for that

Reply:
No need for it I think. Would just make the filter logic more complicated, for little benefit.